### PR TITLE
Add acl to communications, deextrapolate get_path

### PR DESCRIPTION
**Pull request by benvand**
https://trello.com/c/pZNuiwDv/67-communication-files-in-s3-files-are-public

2nd line fix ticket to save communications and clarifications with S3 private ACL
Remove overly extrapolated code
Add tests

### CODE REVIEW

File: app/main/views/communications.py [hunk 1]
```diff
@@ -53,15 +42,19 @@ def upload_communication(framework_slug):
...
 
     if request.files.get('clarification'):
         the_file = request.files['clarification']
         if not file_is_pdf(the_file):
             errors['clarification'] = 'not_pdf'
 
         if 'clarification' not in errors.keys():
-            _save_file(communications_bucket, framework_slug, 'updates/clarifications', the_file, 'clarification')
+            path = "{}/communications/updates/clarifications/{}".format(framework_slug, the_file.filename)
+            communications_bucket.save(path, the_file, download_filename=the_file.filename)
```
Comment (TheDoubleK) [inline app/main/views/communications.py]: I think both should have private ACL maybe? I don't see any reason why we'd have one public and one private.

Comment (benvand) [reply app/main/views/communications.py]: Cool, just looked at the ticket and `agreements` is implied too. Will update when the tests are merged https://github.com/alphagov/digitalmarketplace-admin-frontend/pull/302

Comment (TheDoubleK) [reply app/main/views/communications.py]: I meant both communications and clarifications. Agreements are already private (see https://github.com/alphagov/digitalmarketplace-admin-frontend/blob/master/app/main/views/suppliers.py#L277)

Comment (benvand) [reply app/main/views/communications.py]: Derp, I also this

**Comment by benvand** [comment]
@idavidmcdonald @TheDoubleK 
These are updated now

**Pull request merged**